.github/workflows/web.yml

```diff
@@ -22,6 +22,38 @@ env:
   base_directory: centreon
 
 jobs:
+  changes:
+    runs-on: ubuntu-22.04
+    outputs:
+      has_frontend_changes: ${{ steps.filter.outputs.has_frontend_changes }}
+      has_backend_changes: ${{ steps.filter.outputs.has_backend_changes }}
+    steps:
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            has_frontend_changes:
+              - '**/*.[jt]sx?'
+              - '**/www/front_src/**'
+              - '**/tests/e2e/**'
+              - '**/package*'
+              - '**/lighthouse/**'
+              - '**/tsconfig.json'
+              - '**/cypress/**|**/pnpm-*'
+            has_backend_changes:
+              - '**/*.php'
+              - '**/phpstan.neon'
+              - '**/*.xml|**/bin/**'
+              - '**/tmpl/**'
+              - '**/features/**'
+              - '**/www/src/**'
```

Comment:
```suggestion
              - '**/src/**'
```

Reply:
this will be picked up as a frontend change in @centreon/ui

Reply:
Indeed, I've updated the path to be strict

Reply:
This will take every file changed in `centreon/www/front_src/src`.
We may find another way for this one